### PR TITLE
makes all stats cumulative and then we calculate incremental

### PR DESCRIPTION
**Pull request by edumorlom**
Because most stats on Data Commons are cumulative, in an effort to make the dashboard compatible with other stats, I now calculate the incremental values myself. So i switched to a different stat_vars on config.py.

Also included _difference() function in charge of calculating incremental values. I will add unit tests soon.

ALSO: in config.py, it adds recovered stat_var

### CODE REVIEW

**Comment by beets** [comment]
I'll review this after we get PR #35 merged. There are still outstanding comments there.

**Comment by edumorlom** [comment]
Now instead of having individual /data/Country, /data/County, etc... I have @app.route("/api/data/<string:geo_id>")

File: covid19-dashboard/calculate_data.py [hunk 1]
```diff
@@ -18,6 +18,31 @@
 DateToValueListType = List[Tuple[str, float]]
 
 
+def _difference(date_to_value: DateToValueListType,
```
Comment (beets) [inline covid19-dashboard/calculate_data.py:21]: please add documentation for the function

File: covid19-dashboard/main.py [hunk 2]
```diff
@@ -498,5 +487,13 @@ def add_browser_cache_headers_to_response(data: Dict,
     return response
 
 
+def _place_info_factory(name, contained_in, place_type):
```
Comment (beets) [inline covid19-dashboard/main.py:490]: this is technically not a factory... why not just call it _create_place_info.
also probably worth making it a type.

File: covid19-dashboard/calculate_data.py (hunk 1 above)
Comment (edumorlom) [reply covid19-dashboard/calculate_data.py:21]: fixed.

File: covid19-dashboard/main.py (hunk 2 above)
Comment (edumorlom) [reply covid19-dashboard/main.py:490]: i made it into a class and then do __dict__. is  that ok?

**Comment by edumorlom** [comment]
ready for re-review! thanks

**Pull request merged**